k8_cortx_cloud/logs-cortx-cloud.sh

```diff
@@ -16,13 +16,28 @@ function usage() {
   echo "                                     default file path is ${solution_yaml}."
   echo "    -n|--nodename NODENAME: collects logs from pods running only on given node".
   echo "                            collects logs from all the nodes by default."
+  echo "    --duration DURATION : dutaion for which logs should be collected"
```

Comment:
These options should be added to the usage command line. 
```
echo -e "Usage: \`sh $0 [<optons>]\`\n"
Where options are:
```
Also, please add short options for all parameters.